src/test/operators/export_binary_test.cpp

```diff
@@ -52,12 +52,30 @@ class OperatorsExportBinaryTest : public BaseTest {
   const std::string filename = test_data_path + "export_test.bin";
 };
 
+class OperatorsExportBinaryMultiEncodingTest : public OperatorsExportBinaryTest,
```

Comment:
Haven't looked at the tests in detail yet. Now that we have a parameterized test, do we still need the old tests?

Reply:
the old (OperatorsExportBinaryTests) tests are still used for tests that work with just one encoding type (see FixedStringDictionarySegment or RepeatedIntRunLengthSegment)

Reply:
Got it. However, we should cover all cases (e.g., repeated ints) as part of the parameterized tests, too.